catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -303,14 +297,16 @@
                             ddf.catalog.resourceretriever,
                             ddf.catalog.source.impl,
                             ddf.catalog.util.impl,
-                            org.codice.ddf.catalog.resource.cache,
                             org.codice.ddf.catalog.resource.cache.impl,
+                            org.codice.ddf.catalog.resource.download.impl,
```

Comment:
Why is this being private packaged?  None of the packages included in the project should need this.